luminoth/models/base/base_network.py

```diff
@@ -176,30 +191,10 @@ def _normalize(self, inputs):
         inputs = (inputs - 0.5) * 2.
         return inputs
 
-    def load_weights(self):
-        """
-        Creates operations to load weights from checkpoint for each of the
-        variables defined in the module. It is assumed that all variables
-        of the module are included in the checkpoint but with a different
-        prefix.
-
-        Returns:
-            load_op: Load weights operation or no_op.
-        """
-        if self._config.get('weights') is None and \
-           not self._config.get('download'):
-            return tf.no_op(name='not_loading_base_network')
-
-        if self._config.get('weights') is None:
-            # Download the weights (or used cached) if not specified in the
-            # config file.
-            # Weights are downloaded by default to the $LUMI_HOME folder if
-            # running locally, or to the job bucket if running in Google Cloud.
-
-            # TODO: Shouldn't _config['weights'] be called weights_path or
-            # something similar?
-            self._config['weights'] = get_checkpoint_file(self._architecture)
+    def get_checkpoint_file(self):
+        return get_checkpoint_file(self._architecture)
 
+    def get_base_network_vars(self):
```

Comment:
And doc for this one. What's the difference between this and `get_base_checkpoint_vars`?